css/style.css

```diff
@@ -1 +1,35 @@
+html { 
+  background: url(http://s.hswstatic.com/gif/strength-of-shark-bite-250x150.jpg) no-repeat center center fixed; 
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
 
+
+img {
+	width: 200px;
+	height: auto;
+	border: 3px solid #73AD21;
+	
+}
+
+h3	{ 
```

Comment:
Between the h3 and { you should only have one space. This is the same for all css selectors. This does not affect how the code works but it does make the page much more readable.